package.json

```diff
@@ -38,6 +38,8 @@
     "@ionic/storage": "2.1.3",
     "cordova-android": "^7.1.0",
     "cordova-ios": "4.5.4",
+    "cordova-plugin-aerogear-metrics": "@aerogear/cordova-plugin-aerogear-metrics@~0.3.0",
```

Comment:
I'm seeing a weird issue with these additions

```
npm ERR! Could not install from "@aerogear/cordova-plugin-aerogear-metrics@~0.3.0" as it does not contain a package.json file.
```

If I remove those, everything seems to work fine. Looks like the package-lock.json contained some local stuff that we don't want included.